src/java.desktop/share/native/common/java2d/opengl/OGLBlitLoops.c

```diff
@@ -321,7 +321,8 @@ OGLBlitToSurfaceViaTexture(OGLContext *oglc, SurfaceDataRasInfo *srcInfo,
                 GLvoid *pSrc = PtrCoord(srcInfo->rasBase,
                                         sx, srcInfo->pixelStride,
                                         sy, srcInfo->scanStride);
-                if (slowPath) {
+                                        if (pf != NULL) {
+                                                            if (slowPath) {
```

Comment:
The current indentation does seem to be off.

```c
                GLvoid *pSrc = PtrCoord(srcInfo->rasBase,
                                        sx, srcInfo->pixelStride,
                                        sy, srcInfo->scanStride);
                if (pf) {
                    if (slowPath) {
                        jint tmph = sh;
                        while (tmph > 0) {
...
```

The same applies to the other added if statement.